src/MainWindow.py

```diff
@@ -0,0 +1,106 @@
+from PyQt5.QtWidgets import *
+from src.Connection import DAO
+from win32api import GetSystemMetrics
+from src.TableModel import TableModel
+from PyQt5.QtGui import QPixmap
+from PyQt5.QtCore import Qt
+
+
+class MainWindow(QMainWindow):
+    def __init__(self):
+        super(MainWindow, self).__init__()
+
+        self.setWindowTitle('Application')
+        self.setGeometry(0, 0, GetSystemMetrics(0), GetSystemMetrics(1))
+
+        self.dao = DAO()
+        self.columns = []
+
+        self.table = QTableView(self)
+        self.tableChoice = QComboBox(self)
+        self.widgetLayout = QWidget(self)
+
+        self.addButton = QPushButton('Add record')
+        self.editButton = QPushButton('Edit record')
+
+        self.UI()
+
+        self.tableChoice.currentIndexChanged.connect(self.itemChanged)
+        self.addButton.clicked.connect(self.addButtonClicked)
+
+    def UI(self):
+        buttonLayout = QGridLayout()
+        buttonLayout.addWidget(self.addButton, 0, 0)
+        buttonLayout.addWidget(self.editButton, 1, 0)
+        buttonLayout.addWidget(self.tableChoice, 3, 0)
+        buttonLayout.addWidget(QPushButton('Searching record'), 2, 0)
+        buttonLayout.addWidget(self.table, 0, 1, 8, 8, Qt.AlignTop)
+        buttonLayout.setAlignment(Qt.AlignLeft)
+
+        self.widgetLayout.setLayout(buttonLayout)
+        self.widgetLayout.adjustSize()
+        self.setCentralWidget(self.widgetLayout)
+
+        self.dao.exec("""SELECT table_name FROM information_schema.tables
+                       WHERE table_schema = 'public'""")
+
+        for tableName in self.dao.getFromSelect():
+            self.tableChoice.addItem(tableName[0])
+
+        self.setTable()
+
+    def itemChanged(self):
+        self.setTable()
+
+    def addButtonClicked(self):
+        dialogWin = QDialog()
+        dialogWin.setWindowTitle('Adding window')
```

Comment:
such way isn't good. We need investigate about localization. All work we do is for russian speakers

Reply:
So, should I change all the to ru language?

Reply:
Did it

Reply:
Yup